Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -369,8 +377,8 @@ public void PerformAction(EntityUid performer, ActionsComponent? component, Enti
 
         var toggledBefore = action.Toggled;
 
-        // Note that attached entity is allowed to be null here.
-        if (action.AttachedEntity != null && action.AttachedEntity != performer)
+        // Note that attached entity and attached container are allowed to be null here.
+        if (action.AttachedEntity != null && (action.AttachedEntity != performer && action.Container != null && action.Container != performer))
```

Comment:
Instead of modifying this, just have the action-target entity be a separate field